src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -696,8 +696,129 @@ private Task WriteSuffix()
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private unsafe void EmitHttp11(ref MemoryPoolIterator end)
+        {
+            byte* b;
+            
+            if (end.GetRawBuffer(9, out b))
+            {
+                 // Emit directly
+                *(long*)b = 0x312E312F50545448;
```

Comment:
Instead of hardcoding these numbers, could we generate them from the source string in code, and store the result in a `static readonly` field?  If the perf is the same, I would prefer this for clarity.